static/js/app.js

```diff
@@ -3,45 +3,94 @@ new Vue({
-    },
     methods: {
-        remove: function (event, who) {
+        requestRemove: function (event, who, confirm) {
 
             var vueStuff = this;
 
-            this.$dialog.confirm({
-                message: 'Delete from Queue?',
-                onConfirm: function () {
-                    var bodyFormData = new FormData();
-                    bodyFormData.set('who', who);
-                    axios({
-                        method: 'post',
-                        url: '/api/hands/drop',
-                        data: bodyFormData,
-                        config: { headers: { 'Content-Type': 'application/x-www-form-urlencoded' } }
-                    })
-                        .then(function (response) {
-                            console.log(response);
-                        })
-                        .catch(function (error) {
-                            console.log(error);
-                        });
+            if (confirm == true) {
+                this.$dialog.confirm({
+                    message: 'Remove from Queue?',
+                    onConfirm: () => this.remove(event, who, this)
+                });
+            } else {
+                this.remove(event, who, this);
+            }
+        },
+        remove: function (event, who, context) {
+            var vueStuff = context;
+        
+            var bodyFormData = new FormData();
+            bodyFormData.set('who', who);
+            axios({
+                method: 'post',
+                url: '/api/hands/drop',
+                data: bodyFormData,
+                config: { headers: { 'Content-Type': 'application/x-www-form-urlencoded' } }
+            })
+                .then(function (response) {
+                    console.log(response);
+                })
+                .catch(function (error) {
+                    console.log(error);
+                });
+        },
+        add: function (event, who) {
 
+            var vueStuff = this;
 
-                    var index = vueStuff.handList.indexOf(who);
-                    if (index > -1) {
-                        vueStuff.handList.splice(index, 1);
+            var bodyFormData = new FormData();
+            bodyFormData.set('who', who);
+            axios({
+                method: 'post',
+                url: '/api/hands/rise',
+                data: bodyFormData,
+                config: { headers: { 'Content-Type': 'application/x-www-form-urlencoded' } }
+            })
+                .then(function (response) {
+                    console.log(response);
+                })
+                .catch(function (error) {
+                    console.log(error);
+                });
+        },
+        getHands: function (event) {
+            axios
+            .get('/api/hands')
+            .then(response => (this.handList = response.data.hands))
+            .catch(error => console.log(error))
+        },
+        saveWho: function () {
```

Comment:
rename function to a useful name like `saveLocal` or `saveOnLocalStorage`

Reply:
it does EXACTLY what the name implies, save the name of whoever is using the client